qt-gui.workspace.user.5a83cd3

```diff
@@ -0,0 +1,246 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
Run the command `git rm qt-gui.workspace.user.5a83cd3` to get rid of this file. I don't think this file should be committed to git.